### PR TITLE
Add velocityMap

### PR DESCRIPTION
**Pull request by esclear**
* Automatically converts m/s to a unit that seems fitting
* Also fixed one tests values since technically it should not have worked (Conversion should have happened for velocities >= 0.01c, but that wasn't the case)

Seperate pull request in preperation of a fix for #55.

### CODE REVIEW

File: src/conversion_helper.js [hunk 1]
```diff
@@ -73,6 +73,21 @@ function pressureMap(pa) {
   }
 }
 
+function velocityMap(ms) {
```
Comment (cannawen) [inline src/conversion_helper.js]: Can `ms` be changed to `mPerS` or `metersPerSecond` or something? I first though `ms` stood for milliseconds, and was momentarily confused

File: src/conversion_helper.js [hunk 2]
```diff
@@ -73,6 +73,21 @@ function pressureMap(pa) {
...
+  if (ms <= 56) {
```
Comment (cannawen) [inline src/conversion_helper.js]: Oops, small mistake here! The previous behaviour was "If a number is under 200 mph, only convert to km/h" but 56 m/s is 125 mph (201 km/h, I can see how this bug happened XD). This behaviour is here because people generally have a good grasp of mph -> kmh under 200 mph, but over 200 mph, it might gets harder to visualize so the per-second speed is included as well

Perhaps we an also update the numbers in our tests to test the very boundary cases, instead of random numbers so this type of thing is caught in the future?

**Review by cannawen — CHANGES_REQUESTED** [review]
Hey, great idea to submit this refactor before the main story, it sure makes it easier to review. A small change request, but I love the refactor!

File: src/conversion_helper.js (hunk 2 above)
Comment (esclear) [reply src/conversion_helper.js]: I would dare to say that people who can deal with metric units do have a grasp of velocities <= 200 km/h, but I changed it anyways. More information is better, right?

**Review by esclear — COMMENTED** [review]
I addressed your comments.

File: src/conversion_helper.js (hunk 2 above)
Comment (cannawen) [reply src/conversion_helper.js]: Ah, I was just looking at it purely from a refactoring point of view, I didn't even think of "what it should be." 

But you're right, people generally know that 60 mph ~ 100 kmh and 20 mph is "pretty slow for a car" ... If you think that it makes sense to remove conversions for velocities <= 200km/h, you can create a new github issue and we can discuss further :D It might be a good "first timers only" issue (I'm working on making the repo more newbie-friendly to encourage more people to participate in open source)

**Review by cannawen — APPROVED** [review]

**Comment by cannawen** [comment]
Looks great, thank you!

File: src/conversion_helper.js (hunk 2 above)
Comment (esclear) [reply src/conversion_helper.js]: I agree. Personally I can deal with km/h and m/s no matter what, but if we wanted to change this, this would be the scope of another issue / pull request.
